code/index.html

```diff
@@ -3,28 +3,232 @@
+    <link href="https://fonts.googleapis.com/css2?family=Lexend+Peta&family=Roboto+Mono:wght@400;700&family=Roboto+Slab:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <!-- Found a helpful guide which advocated it's ok to put the JS in the head, as long as you add the "defer" property: https://gomakethings.com/when-should-you-add-the-defer-attribute-to-the-script-element/#:~:text=In%20short%3A%20with%20defer%20on,downloaded%20and%20ready%20to%20run. -->
+    <script src="script.js" defer></script>
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <nav>
+      <div class="nav-content">
+        <div class="nav-logo"><img src="assets/hibr-logo.svg"></div>
+        <div class="nav-links">
+          <a href="#" class="nav-link">What's Hibr™?</a>
+          <a href="#" class="nav-link">Features</a>
+          <a href="#" class="nav-link">Hibr™ a friend</a>
+          <a href="#" class="nav-link">Legal</a>
+          <a href="#" class="nav-link">Guestbook</a>
+        </div>
+      </div>
 
-    <!-- Signup form -->
+    </nav>
+    
+    <header>  
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+      <section class="header-content">
+        <img src="assets/hibr-logo.svg" class="logo">
+        <p>Because right now—we could all use a good year’s sleep.</p>
+        <section class="header-cta">
+          <button class="button-primary teal-bg"><p>Sign up</p></button>
+          <button class="button-secondary teal"><p>Learn more</p></button>
+        </section>
+      </section>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <div class="video-container">
+        <video autoplay muted loop>
+            <source src="videos/sheep.mp4" type="video/mp4" />
+        </video>
+        <h2>Your caption here</h2>
       </div>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    </header>
+
+    <section class="productInfo">
+      <div class="productInfo-content">
+        <div class="hibr-info">
+          <h2>We get it.</h2>
+          <p>What a year, huh? A global pandemic is upon us. The effects of climate change are getting more and more apparent each day. The guy you like <span class="emphasise">still</span> hasn't answered your text. The superpowers of the world are just one slightly too animated tweet away from pressing the nuclear button.</p> <p>We get it.</p><p>It’s more than one person should—nay, could—handle. Sounds like you need to be... Hibr’d™.</p>
+        </div>
+        <div class="hibr-image">
+          <img src="assets/pod.png">
+        </div>
       </div>
     </section>
-    <!-- dont forget to link the javascript file here! -->
+
+    <main>
+
+      <section class="card-full">
+        <h2 class="yellow">Get Hibr'd™</h2>
```

Comment:
Not the best class naming. According to code guidelines we introduced you to during intro days (file in Notion, https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e), better idea would be naming your class something like `card-full-heading`. `yellow` class is not semantic name